test/integration/test_utils.mjs

```diff
@@ -835,6 +835,16 @@ async function cleanupEditing(pages, switcher) {
   }
 }
 
+function normalizeSpaces(d) {
```

Comment:
Please use a slightly more descriptive function name, since "normalizeSpaces" sounds like something that'd be used when parsing regular text.